README.md

```diff
@@ -10,13 +10,64 @@ algebraic systems. Additional modules support input and output as well
 as basic profiling and error management.  
 
 <!---See the [GridPACK home page](https://www.gridpack.org) for more information.-->
+The GridPACK framework will simplify parallel power grid application development
+by eliminating many of the index transformations and calculations required to
+distribute data in parallel applications and also to move data from one
+processor to another as the calculation proceeds. Application writers can focus
+primarily on the physics of their problem without worrying about much of the
+baggage associated with writing applications in parallel. GridPACK supplies core
+functionalities that will simplify parallel code development. These include a
+generic framework for setting up power grid networks and applying whatever
+physical properties to the bus and branches that developers want, functionality
```

Comment:
Can we revert to the previous rather simple introduction to GridPACK? It was for power grid researchers. This one sounds more computer science concentrated.

Reply:
I commented out the extra text. We can move it somewhere else if a good spot can be found for it.